lib/designateDevice.js

```diff
@@ -0,0 +1,23 @@
+const Validator = require('express-validator')
+
+const helpers = require('./helpers')
+
+function handleDesignateDevice(request, response) {
+  const validationErrors = Validator.validationResult(request).formatWith(helpers.formatExpressValidationErrors)
+
+  if (validationErrors.isEmpty()) {
+    const { verificationCode } = request.body
+    const deviceId = request.header('X-API-KEY')
```

Comment:
this line breaks my brain slightly - can you name the variable `apiKey` or something? sorry for being pedantic. and same with the log message below - to me it makes sense that if the device is deciding what the API key should be, then the server shouldn't know or care about how the API key is generated.

Reply:
Done.

Reply:
thank you!